react-native-reanimated.d.ts

```diff
@@ -256,6 +257,10 @@ declare module 'react-native-reanimated' {
       getNode(): ReactNativeScrollView;
     }
     export class Code extends Component<CodeProps> {}
+    export class FlatList extends Component<AnimateProps<FlatList>> {
```

Comment:
@piaskowyk this type definition is missing the generic type mentioned in issue [2769](https://github.com/software-mansion/react-native-reanimated/issues/2769). Should probably be defined as:

```ts
export class FlatList<T = any> extends Component<AnimateProps<FlatListProps<T>>> {
  itemLayoutAnimation: ILayoutAnimationBuilder;
  getNode(): ReactNativeFlatList;
}
```